src/meta/src/rpc/metrics.rs

```diff
@@ -729,6 +732,14 @@ impl MetaMetrics {
         let compaction_event_loop_iteration_latency =
             register_histogram_with_registry!(opts, registry).unwrap();
 
+        let merge_compaction_group_count = register_int_counter_vec_with_registry!(
+            "storage_merge_compaction_group_count",
+            "Count of trigger merge compaction group",
+            &["group_left", "group_right"],
```

Comment:
There will be only one data point per `(group_left, group_right)` label pairs for this metric, which means we will have a lot of inactive timeseries most of the time. We should either have a way to safely remove metrics label via `remove_label_values` or emit the metric with fewer labels.

Reply:
Indeed, I missed the point that the metrics are sufficient to record only the count. There are two points of view
1. merges don't happen very often, and the metrics are more about counts than groups_info. 
2. only `group_left` label is reported, reducing the number of labels

what do you think  @hzxa21?

Reply:
SGTM